src/reducers/grid/observationcontrol/index.js

```diff
@@ -1,5 +1,6 @@
 import Config from '../../../config'
 import { apiUrl } from '../../../util'
+import { sortObject } from '../../../util/sort'
```

Comment:
do this for all containers. remove sort from reducers and place it in containers.

this is example magasin grid container.

```
import { createSelector } from 'reselect'

import { sortObject } from '../../../util/sort'

const getStorageUnits = (state) => state.storageGridUnit.data || []

const getSortedStorageUnits = createSelector(
  [ getStorageUnits ],
  (storageUnits) => sortObject(sortObject(storageUnits, 'name'), 'type')
)
```